fpc_imp_loire_tone.c

```diff
@@ -50,6 +50,8 @@ typedef struct {
     uint32_t auth_id;
 } fpc_data_t;
 
+uint32_t auth_id = 0;
```

Comment:
:-1: on this.
There's already an auth_id field in the fpc_data_t structure that is passed to the function in question, so why would you store it globally instead of in the struct!? (why the one in fpc_data_t isn't used here I'm not sure of!?)

Reply:
Did that intentionally to quickly implement the caching without needing to take care of anything else.
So you're right, the current implementation is wrong. 
Although,storing in the struct may also be suboptimal. Caching in fingerprint.c sounds better, since it would be independent of a specific implementation. What do you think?

Reply:
I suppose fingerprint.c might be the right place, yes.
But it should still be stored in one of the structs.
I spend a lot of time getting rid of global variables, so don't ruin it :P